src/jarabe/journal/listview.py

```diff
@@ -711,6 +716,17 @@ def __drag_data_get_cb(self, widget, context, selection, info, time):
         model = self.tree_view.get_model()
         model.do_drag_data_get(path, selection)
 
+    def __button_press_event_cb(self, widget, event):
```

Comment:
is the button press event the best way to do this?  Is there another event we could use?  Is there like a drag start event?

Does this mean that we will be loading the pixbuf every time I press a mouse button?  That seems expensive.